src/main/groovy/org/grails/gradle/plugin/core/GrailsExtension.groovy

```diff
@@ -43,6 +43,11 @@ class GrailsExtension {
      */
     boolean pathingJar = false
 
+    /**
+     * Whether java.time.* package should be a default import package
+     */
+    boolean javaTime = true
```

Comment:
@matrei Paul [refers](https://groovy.apache.org/blog/groovy-dates-and-times-cheat) to it is as java.time in the groovy documentation

Reply:
All the other names in there are pretty succinct.

Reply:
>  Paul [refers](https://groovy.apache.org/blog/groovy-dates-and-times-cheat) to it is as java.time in the groovy documentation

I would guess he is using the package name. He is also using classes from the subpackages when referencing `java.time`. 

There is also this (which I think is the more common name):
> Since Java 8, the JDK has included the [JSR-310](https://jcp.org/en/jsr/detail?id=310) Date Time API. We refer to these classes as the new date classes.

Reply:
> All the other names in there are pretty succinct.

I know...

Reply:
@matrei  so importJavaTime ok or do you want importDateTime or  importDateTimeApi?
I think autoImportDateTimeApi is too long compared to other properties.